### ISSUE TITLE
[iOS] Fix helper text a11y

### ISSUE THREAD

**Issue by edschema** [openedx/wg-mobile#42]
Form entry helper text does not meet a11y contrast requirements. This is high priority. 

Update: this was an issue with textSecondary that has other downstream implications: textSecondary and textSecondaryLight both resolve to the same color. Icons and other non-text elements are using textSecondary.

**Comment by edschema** [comment]
Opened PR https://github.com/openedx/openedx-app-ios/pull/658 to fix this issue. This ended up being a bit more complicated than changing the helper text token:
- helper text is defined by textSecondary which did not have sufficient contrast
- there are many instances of textSecondary text that is used to define color for things it should not: icons, placeholder text
- textSecondary and textSecondaryLight are both resolve to the exact same color: I did not fix this as it should be fixed as part of a larger color token effort
